nodes/stampzilla-knx/tunnel.go

```diff
@@ -15,21 +16,26 @@ import (
 )
 
 type tunnel struct {
-	Node      *node.Node
```

Comment:
Also aligned this struct for memory effiency (vim lints about that for me now days... ) 
https://medium.com/@felipedutratine/how-to-organize-the-go-struct-in-order-to-save-memory-c78afcf59ec2